chapter-1/bins/chap1.c

```diff
@@ -9,6 +10,8 @@ char* formulaInput;
 int main(int argc, char** argv) {
   // Read filename from args
   char* filename = argv[1];
+  (void)filename;
+  (void)argc;
```

Comment:
What's going on here?

Reply:
I turned on `-Wall -Wextra -pendantic` to give extra compiler warnings, and `-Werror` to turn warnings into errors. Unused args are warned against, and for better or worse this is one way to circumvent that warning when variables/arguments need to be present but aren't used.

I'd prefer to figure out how to turn that particular warning off (for procedure arguments) but didn't figure out how yet. We could do something like

```
#define DECLARE_UNUSED(var) (void)(var)
```

to make the intent clearer if you prefer.

Reply:
Not too fussed, just asking out of curiosity 😄